troposphere/static/js/components/modals/allocationSource/NoAllocationSourceModal.react.js

```diff
@@ -52,8 +52,11 @@ export default React.createClass({
 
     renderInstanceList: function (project) {
         return this.props.instances.map((instance, index) => {
+            let projectId = project.id;
+            let instanceProjId = instance.get('projects')[0];
 
-            if (true) {
+            if (instanceProjId === projectId) {
```

Comment:
@cdosborn I know we had talked about not doing it this way. But I wanted to at least have it as a comparison since it is so simple, I don't don't think the goofy index bit is reason enough for a long winded alt solution.